### PR TITLE
Add parameter -d

### LINKED ISSUE #2: Parameter -d

### PR DESCRIPTION
**Pull request by justinmueller05082000**
fixes #2

### CODE REVIEW

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Rebase against the (pulled) master and resolve conflicts.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git rebase -i HEAD~2` (In the appearing editor swap the two lines starting with "pick".)
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD feature/add-copying-file-9)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/parameter-d-2`

File: main.go [hunk 1]
```diff
@@ -92,4 +98,4 @@ func main() {
 			break
 		}
 	}
-}
+}
```
Comment (Al2Klimov) [inline main.go]: Not part of the task.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD feature/add-copying-file-9)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/parameter-d-2`

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Write a test protocol.

**Comment by justinmueller05082000** [comment]
**Test protocol:**

**toCut.txt:**
```
abcdef:ghiklmno-linuxabc
drg:dogktnc-skgiz
vmtas.ieps:apld-kfma
```

**Go command:**

**Terminal:** `go run . -d : -f 1 < toCut.txt`

**Output:**
```
abcdef
drg
vmtas.ieps
```

**Cut command:**

**Terminal:** `cut -d : -f 1 < toCut.txt`

**Output:**
```
abcdef
drg
vmtas.ieps
```

**Conclusion:**
Both results are equal.

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**